public/js/app/routes/PostRoute.js

```diff
@@ -1,8 +1,8 @@
 define(["app/app",
-        "ember"], function(App, Emberx) {
+        "ember", "components/TransitionalRoute"], function(App, Ember) {
```

Comment:
More nitpicking on code style: maybe it's better to put every dependency on a new line?